homeassistant/components/fronius/quality_scale.yaml

```diff
@@ -0,0 +1,79 @@
+rules:
+  # Bronze
+  action-setup:
+    status: exempt
+    comment: |
+      This integration does not provide additional actions.
+  appropriate-polling: done
+  brands: done
+  common-modules: done
```

Comment:
exempt, only one platform

Reply:
Yes, there is no common entity class, but there is still a a coordinator stored in coordinator.py. I guess that's whats requested by https://developers.home-assistant.io/docs/core/integration-quality-scale/rules/common-modules

Reply:
imo it asks for both, that's why I voted for exempt

Reply:
I've added a comment explaining why `entity.py` is not needed in this integration.